src/components/Users/UserAvailabilityTab.tsx

```diff
@@ -163,12 +174,14 @@ export default function UserAvailabilityTab({ userData: user }: Props) {
                   <div />
                 </div>
               </PopoverTrigger>
-              <DayDetailsPopover
-                date={date}
-                templates={templates}
-                unavailableExceptions={unavailableExceptions}
-                setQParams={setQParams}
-              />
+              {availableUserId && (
+                <DayDetailsPopover
+                  date={date}
+                  templates={templates}
+                  unavailableExceptions={unavailableExceptions}
+                  setQParams={setQParams}
+                />
+              )}
```

Comment:
wouldn't this still make the mouse pointer to be hand on the calendar? 🤔 although the popover won't open.

let's instead disable the popover trigger itself directly